code/game/data_huds.dm

```diff
@@ -189,6 +189,7 @@
 	else if(stat == DEAD || (HAS_TRAIT(src, TRAIT_FAKEDEATH)))
 		if(tod)
 			var/tdelta = round(world.time - timeofdeath)
+//			var/obj/item/organ/heart = getorganslot(ORGAN_SLOT_HEART)
```

Comment:
Goal is to remove the "huddefib" when either the heart or brain is failing, but for whatever reason it wasn't updating when we check against them

Reply:
That goal will have to wait!

Reply:
commented code cringe